framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -1576,8 +1622,8 @@ private void handleExpr(AnnotationBuilder builder, String name, Expression expr)
         }
         TypeElement rcvElt = elements.getTypeElement(faexpr.getScope().toString());
         if (rcvElt == null) {
-            // Search imports for full annotation name.
-            for (String imp : imports) {
+            // Search importedConstants for full annotation name.
+            for (String imp : importedConstants) {
                 String[] import_delimited = imp.split("\\.");
```

Comment:
Above on 1589 a `StubUtil.partitionQualifiedName` helper method is used. Would it be easier to do the same here, instead of splitting the String?

Reply:
I'm not familiar with this code.  I only renamed the constant here, so I'm not sure.